src/error.rs

```diff
@@ -1,5 +1,6 @@
 //! Rust types for working with Ruby Exceptions and other interrupts.
 
+use core::ffi::c_char;
 use std::{any::Any, borrow::Cow, ffi::CString, fmt, mem::transmute, ops::Deref, os::raw::c_int};
```

Comment:
Could you add `c_char` to the end of the `use std::…` line, like `…, os::raw::{c_char, c_int}};` instead.

`core::ffi::c_char` was nightly only up until the latest release of Rust, whereas `std::os::raw::c_char` has been available for ages (even though it’s just a reexport of `core::ffi::c_char`), and I’d like to keep things working on older Rust versions.

Reply:
Fixed.